src/CowCareComponent/rainy.jsx

```diff
@@ -0,0 +1,76 @@
+                    <h3>Foot Care:</h3>
+                    <div>Mud and wet conditions can lead to hoof problems. Regular hoof trimming helps prevent issues like foot rot.
+                    Ensure proper drainage in pens and paddocks to reduce mud accumulation.</div>
+                    <h3>Disease Prevention:</h3>
+                    <div>Vaccinate your cows against common rainy season diseases (see below).
+                    Implement a herd health program with the guidance of a veterinarian.</div>
+                    <h3>Nutrition:</h3>
+                    <div>Adjust the diet to account for changes in forage quality during the rainy season.
+                    Monitor body condition and provide necessary supplements.</div>
+                    <h3>Water Management:</h3>
+                    <div>Ensure access to clean and uncontaminated water sources.
+                    Prevent waterlogging in feeding and resting areas.</div>
+                    <h2>Common Rainy Season Diseases in Cattle:</h2>
+                    <h3>Foot Rot:</h3>
+                    <div>Caused by bacteria in moist conditions.
+                    Keep areas as dry as possible, and promptly treat any signs of lameness.</div>
+                    <h3>Respiratory Infections:</h3>
+                    <div>Increased moisture can contribute to respiratory issues.
+                    Provide good ventilation in barns and address any signs of coughing or nasal discharge.</div>
+                    <h3>Internal Parasites:</h3>
+                    <div>Wet conditions favor the survival of parasites.
+                    Maintain a deworming program based on veterinary recommendations.</div>
+                    <h2>
+                    Vaccines for Cattle During Rainy Season:
+                    </h2>
+                    <h3>Clostridial Vaccines:</h3>
+                    <div>Protect against diseases like blackleg.
+                    Administer booster shots as recommended by your veterinarian.</div>
+                    <h3>Foot-and-Mouth Disease (FMD) Vaccine:</h3>
+                    <div>FMD is more prevalent in wet conditions.
+                    Vaccinate according to the recommended schedule.</div>
+                    <h3>Leptospirosis Vaccine:</h3>
+                    <div>Leptospirosis can spread through contaminated water.
+                    Consider vaccination to prevent leptospirosis.</div>
+                    <h3>Bovine Respiratory Disease (BRD) Vaccines:</h3>
+                    <div>Vaccines targeting common respiratory pathogens.
+                    Especially important in areas prone to respiratory infections.</div>
+                    <h2>General Disease Prevention Measures:</h2>
+                    <h3>Regular Veterinary Check-ups:</h3>
+                    <div>Schedule regular visits with a veterinarian for health assessments and vaccinations.
+                    </div>
+                    <div>Quarantine New Animals:
+
```

Comment:
pull all constant content to const file and use those variables here

Reply:
how to to do this I am not understand